src/qt/glcanvas.cpp

```diff
@@ -378,6 +378,15 @@ wxGLCanvas::wxGLCanvas(wxWindow *parent,
     Create(parent, id, pos, size, style, name, attribList, palette);
 }
 
+wxGLCanvas::~wxGLCanvas()
+{
+        // Avoid sending further signals (i.e. if deleting the current page)
+        m_qtWindow->blockSignals(true);
```

Comment:
Another style comment: indentation seems wrong here, i.e.
```suggestion
    // Avoid sending further signals (i.e. if deleting the current page)
    m_qtWindow->blockSignals(true);
```

Reply:
Agreed. Will fix.